spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Contestants Index Page' do
+  describe "index page has list of all contestants names and projects" do
+    it "can see a list of all contestants names and projects" do
+      recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+      furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+
+      news_chic = recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+      upholstery_tux = furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+      
+      jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+      gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+
+      ContestantProject.create(contestant_id: jay.id, project_id: news_chic.id)
+      ContestantProject.create(contestant_id: gretchen.id, project_id: upholstery_tux.id)
+      # As a visitor,
+      # When I visit the contestants index page ("/contestants")
+      visit "/contestants"
+      # I see a list of names of all the contestants
+      expect(page).to have_content("Jay McCarroll")
+      expect(page).to have_content("Gretchen Jones")
+      # And under each contestants name I see a list of the projects (names) that they've been on
+      expect(page).to have_content("News Chic")
+      expect(page).to have_content("Upholstery Tuxedo")
+      # (e.g.   Kentaro Kameyama
+      #         Projects: Litfit, Rug Tuxedo
```

Comment:
I would get into the habit of using `within` blocks to test where something should be showing up on the page. So you may want to break this up into two tests - one for `names of all contestants` and one for `list of projects under each contestant`. It could look something like:
```ruby
it "I see a list of names of all the contestants" do
    visit "/contestants"

 expect(page).to have_content("Contestants")
 within "#contestants_list" do
    expect(page).to have_content(@jay.name)
    expect(page).to have_content(@gretchen.name)
    expect(page).to have_content(@kentaro.name)
    expect(page).to have_content(@erin.name)
  end
end   

it "And under each contestant's name I see a list of the projects (names) that they've been on" do
            visit "/contestants"
within "#contestants_list" do
  within "#contestant_#{@jay.id}" do
    expect(page).to have_content("News Chic")
    expect(page).to_not have_content("Upholstery Tuxedo")
    expect(page).to_not have_content("Boardfit")
    expect(page).to_not have_content("Litfit")
   end

   within "#contestant_#{@gretchen.id}" do
     expect(page).to have_content("News Chic and Upholstery Tuxedo")
     expect(page).to_not have_content("Boardfit")
     expect(page).to_not have_content("Litfit")
    end

     within "#contestant_#{@kentaro.id}" do
      expect(page).to have_content("Boardfit and Upholstery Tuxedo")
      expect(page).to_not have_content("News Chic")
      expect(page).to_not have_content("Litfit")
     end

      within "#contestant_#{@erin.id}" do
       expect(page).to have_content("Boardfit")
       expect(page).to_not have_content("News Chic")
       expect(page).to_not have_content("Litfit")
       expect(page).to_not have_content("Upholstery Tuxedo")
      end
end